docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.9"
 services:
   ddd-autohaus:
-    image: ghcr.io/dqualizer/dqsamples-ddd-autohaus
+    image: ghcr.io/dqualizer/ddd-autohaus:main
```

Comment:
Das neue Image lässt sich nicht ziehen (Unauthorized)

Reply:
 Same for me, when I try to pull separatly.

Reply:
Die Visibility war noch auf private, anscheinend bin ich Lokal noch angemeldet und deswegen hat es funktioniert :D. Hab die Visibility auf Public gestellt 👍